frontend/src/components/common/table-placeholder/table-placeholder.tsx

```diff
@@ -0,0 +1,9 @@
+import { ReactElement } from 'react';
+
+import styles from './styles.module.scss';
+
+const TablePlaceholder = (): ReactElement => {
+  return <div className={styles.placeholder}>No data to display</div>;
```

Comment:
Maybe it's better to wrap text in span? Cause div is a block tag

Reply:
But Vlada can use paragraph (p) as well 🤔

Reply:
Paragraph sounds better